src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/Main.java

```diff
@@ -157,6 +158,15 @@ private static void processJavaSourcesJar(File inputFile, APIListing apiListing)
         apiListing.setLanguage("Java");
         apiListing.setMavenPom(reviewProperties.getMavenPom());
 
+        if(packageName.indexOf("spring") > -1) {
+            apiListing.setVariant(LanguageVariant.Spring);
+        } else if(packageName.indexOf("android") > -1) {
+            apiListing.setVariant(LanguageVariant.Android);
+        } else {
+            apiListing.setVariant(LanguageVariant.Default);
+        }
+        System.out.println("  Using '" + apiListing.getVariant() + "' for the language variant");
```

Comment:
Eventually we will want a smarter way of discerning this, but it should do for now